### ISSUE TITLE
Modpack news

### ISSUE THREAD

**Issue by odenthefish** [Xariyl/FishyExperiment812390#9]
News for the modpack

**Comment by odenthefish** [comment]
Due to a technical issue (on my side) it will be a bit longer to the publishing 
But i will probably upload a version without journeymap on the technic launcher in the meantime

**Comment by odenthefish** [comment]
There will be an update tonight but ignore it! It is just getting a few things ready for being made public

**Comment by odenthefish** [comment]
I have submitted the modpack! after so long I have finally done it! yay! now we need to wait for it to be approved! wooo! there will be an update pretty soon as well but probably not tonight.

**Comment by Xariyl** [comment]
Yay!

Need me to make any additional updates to the server?

**Comment by odenthefish** [comment]
Not yet I still haven't updated the mods

**Comment by odenthefish** [comment]
I thought "hey it will be easy to make a curse voice modpack! I know how to do Technic modpacks so this can't be so hard!" and after reading the modpack submission guidelines.... It is more complex than physics! It wants a manifest.json? It says that 3D party mods not on the list are not accepted or something strange, some of the bigger mods in the modpack are not on the list so that is not an option.
Unless I missed something it isn't an option for the modpack, at least at the moment.

**Comment by odenthefish** [comment]
there should be an update "soon" it'll before the pack becomes public for sure but here is some teasers! 
http://imgur.com/a/GLOVu

**Comment by Xariyl** [comment]
That looks like Traveller's Gear, yes? Cool little mod, I always liked it :)  Has compatibility with Mariculture also, right?

**Comment by odenthefish** [comment]
Yes and it has compatibility with mariculture like you said extensive support for baubles and some pretty good support for tinker's gear 
along with that aroma's mining world mod will be included and thermal smeltery

**Comment by odenthefish** [comment]
Update incoming! it should be a little while because my internet is so bad... But it adds damage indicators, travelers gear, thermal smeltery and aroma's mining world (Wooo!). 
it'll also add/fix 3 NPCs (Jake, Police and new NPC Chris the guard)
Configs are updated for the new updates (namely turrets)
it updates a silly amount of mods (somewhere in the way of 30)

**Comment by Xariyl** [comment]
Great! Will update the server as soon as I can, then.

**Comment by odenthefish** [comment]
Update! V1.whoknows is out!

**Comment by odenthefish** [comment]
I'll start working on getting the pack public on the ATlauncher, I'm really dreading making up the licence sheet...
Also I will be out of town the next 2-3-4 days, I will be able to work on getting the modpack public in some of that time.

**Comment by Xariyl** [comment]
I'll be out of town also, actually, leaving Wednesday and back on Sunday. So no rush.

**Comment by Xariyl** [comment]
modpack v1.0 has been uploaded to the server

**Comment by odenthefish** [comment]
Whee!

**Comment by odenthefish** [comment]
Inventory issues are thought to have ceased, work on publication will resume.

**Comment by odenthefish** [comment]
I finally have another optical mouse, work will actually resume.
Sorry about all the putting-off I've done for getting it public, I have been ether really busy or the internet is so bad I can't do anything.
It should be in the next few days its public.

**Comment by odenthefish** [comment]
Double checking permissions, The pack will be submitted soon.

**Comment by odenthefish** [comment]
Updating the modpack, but don't update anything until later, I still need to do a few tests and it won't be done uploading until tomorrow night because I have a messed up schedule and I don't like leaving my laptop on at night. 
What if i told you that thermal smeltery was installed on the server but not on the client? 
I thought that forge caught things like that, ether as missing blocks or items... Looks like we have our inventory ick culprit.

**Comment by odenthefish** [comment]
Modpack has been uploaded, its quite amazing that it takes over 2 hours to sync at my house and less than 10 minutes to sync at my workplace...
I haven't updated forge or tested on a local server (I've been trying to always test ever since the simply jetpacks thing...)

**Comment by odenthefish** [comment]
It looks like everything is ok, but several mods have updated in the time it took to upload the file... although that was like 2 days....
But make sure to back everything up configs, player data ect. because when I booted up the test server it had a bunch of messed up things like iron railcraft lanterns in the place of some kind of stone
I did delete a bunch of stuff but its something to be careful about.
http://imgur.com/a/6tBfq
So the modpack is ready when you are!

**Comment by Xariyl** [comment]
Wow... looks like some ID's got corrupted.

Okay, I'll download and test a bit when I get home from work tonight.

**Comment by odenthefish** [comment]
So how is the update going? On another note I'm getting ready to go public!

**Comment by Xariyl** [comment]
Sorry, I haven't had a chance to test things out this week. I've got a copy of the server world that I wanted to throw into single player and make sure there's no conflicts or corruption. I haven't gotten to it yet. I'll try tonight to put some time to it.

**Comment by odenthefish** [comment]
Ok,  I understand not having time or Internet connection to do stuff.

**Comment by Xariyl** [comment]
Server files updated with the 8/31 modpack. So far no corruption noted, no mismatched blocks or any other significant issues. Looks like everything is okay. If you have any issues let me know :)

**Comment by odenthefish** [comment]
Woo! I thought it had something to do with me deleting all the config and replacing them...

**Comment by Xariyl** [comment]
I replaced the configs in my test world and didn't see any issues. I left the ones on the server alone though since I have some custom stuff in there.

**Comment by odenthefish** [comment]
So make since = null 
I don't know what happened, it must have been a one time thing.

**Comment by Xariyl** [comment]
Yeah, I don't get it.  I did a quick fly around the world looking for something, was on for about half an hour, and didn't see any problems.

Your computer is fun :)

**Comment by odenthefish** [comment]
It is "fun" alright :P

**Comment by odenthefish** [comment]
owh,so if you can be updated experiment 812390?

You had to be there.....
Anywho... modpack has been updated with the latest versions of a few mods.
It updated ender tech, botania, Malisis core and doors, buidcraft, pnuematic craft and modular power suits.
Perhaps updating modular power suits will fix my problem... I've been breaking all the things and you've been fine! I'm jelly fish.
I thought I tested the modpack on a local server and put on a power suit without issue... Must have forgot to...

**Comment by odenthefish** [comment]
I have actually finished double checking the permissions!!! The pack SHOULD be public today, internet permitting... There will be an update before hand and the pack will probably be moved to google drive due to it having soo much more default room than dropbox. (FTB wants to keep the archived versions as well... are they the IRS? :P )

**Comment by odenthefish** [comment]
WARNING: Update inbound! Take cover!
Changelog:
Mods:
Updated:
Carpenter's blocks
Forestry
Open modular turrets
bibliocraft
botania
ender tech
gardenstuff
malisis doors
modular powersuits
pneumaticcraft
Numinia
Config:
Deleted:
Machinemuse recipe due to possible console spamming.

ATG hasn't been added yet, it will added next update.
The update will be posted in google drive.
client https://drive.google.com/file/d/0B7SE6kZCr1InN0t6cTk4d1puUGc/view?usp=sharing
server https://drive.google.com/file/d/0B7SE6kZCr1InR2lDek5Cb2pYelU/view?usp=sharing

**Comment by odenthefish** [comment]
Removed fastcraft: fastcraft is so much trouble that I didn't want to chance the licence.
But I will put something telling that fastcraft is recommended.

**Comment by odenthefish** [comment]
SUBMITTED!!! It might take up to 72HRs for it to be up but YAY! We have a modpack!

**Comment by Xariyl** [comment]
Yay!

I'm out of town for the weekend but will update the server when I get home.

Regarding Fastcraft:

```
TL;DR: Only for mod packs, unmodified, no pay or ad wall, warning+link+author+bug report mentioning in the description, disableable+upgradeable in launchers, update within 3 weeks, conditions may change.

Example description by Slowpoke101:
 Quoted
This pack contains Fastcraft, by Player, enabled by default. Fastcraft enhances Minecraft with increased performance. Bug reports being made directly to Mod Authors should state Fastcraft is enabled.
```

So IF you want to include it, you can.

Once everything is finalized I'll clear out all these dev-issues as you suggested before. Think we'll want a world reset?

**Comment by odenthefish** [comment]
Nah, really the only things that have changed concerning world gen is the Rouge dungeons and battle towers and that was a bit ago.
I wouldn't recommend ATG for a server because the local client-integrated forge server was getting horrible TPS lag even down to 3% TPS at times, just imagine that on a server and people like me with Internet slower than a dead snail, though for people with very good hardware it'd be great.

**Comment by odenthefish** [comment]
Small status update with no real impact on you:  I thought that FTB wanted me to keep all versions of the modpack! (Silly I know but people don't make since) but it turns out that I read that completely wrong...

Updating your pack:
Reply to your thread, which can be found here once you have submitted the pack, with a link to the complete new zip. The images should not be included unless they have changed. You can change your pack code at infrequent intervals, but the old code will remain available. However, only the new code will update.

You cannot update only the server, forge version, images, description or any other pack details. These must be updated with a full pack update.

Give in your message any updated details about the pack, new description, any new permissions, new forge version, etc.

A 3rd party team member will update your pack within 72 hours and respond once they have done so.

I read the code thing wrong... :P

**Comment by Xariyl** [comment]
Probably a good idea to keep a couple versions available, in case a bug is found in a newer version and people want to roll back until fixed.

They do make the update method seem cumbersome, though. But still... very cool that it's almost ready. Congrats :)

**Comment by odenthefish** [comment]
Thanks, I couldn't have done it without your help :) really it made it a lot easier with your help :) 
Also my cat is snoring...

**Comment by odenthefish** [comment]
Houston we have a modpack.
Code: Experiment812390
I kind of forgot the Logo though... It'll be added next update.. and that it coming soon!
I will get to making a thread in a bit...
Server: https://drive.google.com/open?id=0B7SE6kZCr1InR2lDek5Cb2pYelU

**Comment by odenthefish** [comment]
Xariyl? Are you dead Rasta?

**Comment by Xariyl** [comment]
I'm somewhat around. Been very busy lately and haven't had much chance to get online. Not for more than a few minutes at a time, anyway.

But still alive :)

**Comment by Xariyl** [comment]
Weird, for some reason ProjectRed couldn't automatically download the latest MrTJPCore file. I've downloaded the latest version from Curse (1.1.0.31) and manually loaded it to the server, which then immediately crashed. So I restored version 1.0.9.22 from a backup, and now everything seems to be working okay.

I modified the config for Battletowers on the server a bit, so that they're a bit more rare; they spawned way to frequently for my taste. I also turned off the decay of the Botania flowers (so that Dayblooms/Nightshades/Hydranias don't disappear). 

If you feel anything else needs tweaking, let me know.

**Comment by odenthefish** [comment]
Yeah, I've been using an old ver

**Comment by odenthefish** [comment]
_Feels like an idiot for pressing the close and comment button while typing_
I've been using an outdated version of PJ for a long time now, I believe that it is time to try and update (I haven't due to it being very buggy, I still remember the -1 Stonebricks) 

I felt like the Battle towers are a bit common as well but I kept forgetting to change that.

I didn't think about the config for decay, that is a nerf that gets on my nerves a bit...

**Comment by odenthefish** [comment]
Update sample:
Alternate terrain generation
Changed config for custom NPCs so that non-Ops can spawn in NPCs (Only on client side, to make sure you can spawn them without cheats on, I'm not sure if it keeps you from it in SP)
Changed battletowers config so they don't spawn so much (Was 196 blocks between towers now 1500)

**Comment by odenthefish** [comment]
The muddy pack has been updated!
It'll take a bit for the FTB people to update it but here is the server link for when it goes live https://drive.google.com/open?id=0B7SE6kZCr1IneHF0dDNyZTRRLWc
Changelog:
Updated
Aroma core
bibliocraft
chisel 2
ender tech
forestry
garden stuffz
IGW
pneumaticcraft
schematica
buildcraft 
botania 
BC compat
I didn't update project red again because when I did it crashed because of an enchantment ID being the same as a mariculture, I'll update it next update I promise! I've been sticking with this version because it seems to have several bugs and this version works well.

Added Alternate terrain generation!!!! This mod is just calling for an adventure LetsPlay....

On the config side of things I've done this stuff...
Added biome config for ATG 
Changed client side Custom NPC's config to allow non-OPs to spawn NPCs
Changed battletowers config so they don't spawn so much (Was 196 blocks between towers now 1500)

**Comment by Xariyl** [comment]
Sweet! I'll update the server soon(ish).

**Comment by odenthefish** [comment]
Ok, the pack hasn't been updated yet (the ftb people are the ones who update the modpack once you submit the update so it conflicts with my schedule :P ) it'll probably be Monday or so that the update goes live :)

**Comment by odenthefish** [comment]
Yay! the modpack has been updated! And I remembered to put artwork! in the wrong folder! heh 
It'll come next update, you can't change certain things unless it comes with a full update, and the image is one of them.
I'm guessing that they put that in so people aren't spamming tiny updates.

**Comment by Xariyl** [comment]
Nice! Server updated this morning. I only had a few minutes to check things out but no issues noted so far.

**Comment by odenthefish** [comment]
What will be updated in the next update: 
+Finally remove XP cost from some open blocks things (elevator and probably some other stuff) 
+Disable withering of botania daybloom, nightshade, hydrograinga (hydrodraingdasfwe? whatever :P )
-Removed Xarbot 2000.
:P I think I'll start adding silly things in the update changelog (probably along with a proper changelog)

**Comment by Xariyl** [comment]
Update # 67:  Due to popular request, Rad the Bard will now include random death metal music and disco songs in his playlist. The Death Metal Disco Duck is an original, and reproduction may violate copyright laws in your country.

Update # 69: Xarbot was found to be too sexy, and was distracting Chrisbot so much that mobs invaded the Spawn castle. A long hair will be added extruding from Xarbot's left nostril to de-sexy-ify.

Update # -60125:  The real Xariyl finally trimmed that left nose hair that was weirding everyone out. Thanks for all the bug reports on this, don't know how it made it out of development intact.

Bug report #726391: Xariyl is way too easily amused with himself. Often seen at the railroad tracks to the End without pants and giggling to himself.

**Comment by odenthefish** [comment]
LOL

**Comment by odenthefish** [comment]
Time scaler doesn't seem to be updated anymore, he keeps referencing Toomuchtime mod http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/2145092-forge-toomuchtime-v2-5-0-control-over-minecrafts what do you think about using it instead?

**Comment by Xariyl** [comment]
I looked at it, and that's fine. DMiller does good work.

Is this something you want in the actual pack, or want me to test on the server first?

**Comment by odenthefish** [comment]
I'll add it to the pack, it seems like a good idea.

**Comment by odenthefish** [comment]
Update snapshot: 
Updated mods:
Botaina
Buildy craft
Buildy craft compat
Endy tech
Forestry
Mudular powersuits
Pneumatic craft
Hurdcore darkeness
Open muddy turrets
Travelerr's gears

Updated, but not all the way up to date: Forestry was causing Biomes O' Plenty to crash somehow, all the blame was on BOP but it stopped crashing when I downgraded Forestry a version, probably an API
Morpheus says it's outdated in the mod version checker but I checked on the curse page and it is up to date, probably it is detecting a beta build or something.

Mods not added this update: Forcaster's backpacks ( I'll give it a bit more time, it seemed a bit buggy)
Pam's harvest craft (I'm waiting for Agriculture to update, it adds better elements of cooking at least in my book)
the Ships mod, it'll probably be added in another update, I don't like introducing too many mods in one update (it makes it easier to isolate any new bugs) and I haven't had enough time to test the ships mod, 
and finally Big doors I might be added later, it seems quite buggy at the moment.

Mods added Toomuch time, Extra TIC

A config snapshot will come soon.

**Comment by Xariyl** [comment]
"Open muddy turrets"... I have a vision of Gany's poop-mod, loaded into the turrets and firing feces at mobs. I might have to do that in single player someday :)

Agriculture... is that the AgriCraft mod? It adds crop breeding like IC2 but more in depth and advanced. I played with it a bit on the Agrarian Skies 2 pack and enjoyed it. It could be a fun addition.
In that vein, what do you think of AppleCore? It adds the underlay to the hunger bar so you can see your saturation values. It's client side, though, I've been using it a few days with no issues.

Ships looks good though, I'd like to see that one in. I wonder if we can create vanilla TNT canons and work them on the ships!

**Comment by odenthefish** [comment]
:P
Agriculture is a mod that adds things like ovens and other kitchen appliances that are used to make food, sort of like pam's harvest craft but more complex, though I'm not sure if it will ever be updated (more food types were requested by Sandriell, but I'm not a big fan of harvest craft, it seems too big for me)

I've been eying agricraft for a while, it looks fun and making you get 10/10/10 on 30 plants seems fun, ya know just to make sure it's all working :P 

Looking at apple core that looks very useful, I've always wondered why minecraft doesn't show saturation

Ships mod still might make this update, the update was meant to go up friday, then Sunday but I never had time those days :P ( if you are vundering yes with a v why those specific days it Is due to those days having quick internet.)

Conclusion: agricraft probably next update, apple core this update probably, and ships mod might make it in this update.

**Comment by odenthefish** [comment]
TooMuchTime mod probably won't make it in this update, the reason is that when I was testing (playing with the ships mod for over an hour) I kept seeing the sky suddenly turn dark, I believe this is the time mod, maybe I configured it wrong but I'm going to keep trouble shooting to see if I can find out if it really is toomuchtime doing it.
It is at about dusk, and the sun suddenly turns into the moon and it's night.

**Comment by odenthefish** [comment]
Hmm, The modpack is behaving strangely, I keep getting crashes on the local server when pneumatic craft is installed and when I try and connect to my local server without PC I keep getting rejected...
The modpack will have a slight delay.

**Comment by Xariyl** [comment]
When I was testing TimeScaler, I used the config options:

```
timescale = 0.33333, 2
timezone = local
offset = 0
```

This should make the day time last twice as long, but the night half as long.

For the crashing; doesn't PC add seeds/plants of some sort? I wonder if it's crashing with Agriculture or AgriCraft?

**Comment by odenthefish** [comment]
PC used to use seeds, but they are being phased out for the use of oil to make plastic, agriculture and agricraft aren't in the pack yet.
The crash log is complaining of a GUI error or something, I'm going to try reinstalling my local server, I've had it around a while and something might have corrupted the forge files.

**Comment by odenthefish** [comment]
Woo! The crashing I was having was a known issue and it has now been fixed, The modpack should be submitted soon, It includes applecore, extra tic, and the ships mod, I'd like to experiment more with toomuchtime a bit more, we could put it on the server in the mean time.

All I have to resolve is the reason mushroom islands keep spawning connected to land every world I create.....

**Comment by odenthefish** [comment]
WARNING: Your device health is at critical, download new modpack update to save your device.
11 updates detected! 
3 additions detected! 
1 server update detected! 
1 new artwork for the modpack detected! 
2 Config changes!

Sorry, I couldn't resist doing a classic "Your device's heath is critical" style update list.

Anyway, new mods are Applecore, Extra TiC and Cuchaz's ships mod
The config changes are that now you can't pickpocket people's enderchests with a hand of ender, That is a cool feature but I feel that vanilla enderchests should be able to safely store your items without the threat of being robbed. (like how they are commonly used on servers)
and now passive flowers _might_ not wither, that or they instantly wither now. They didn't when I was testing the change but who knows, My laptop is crazy.
I sat the ticks before they wither to 0 so hopefully that disables that. 
EDIT: I think that the config is broken... I sat the ticks to 2 ticks before they wither on a single player world and they haven't withered even after 10 minuets....
The modpack isn't uploaded yet, but I submitted it to FTB so it won't be very long.
And when you update the modpack on the server make sure to back the server up, I feel like there is something I didn't do right but I can't find anything....

**Comment by odenthefish** [comment]
Warning! An update to the modpack came out today, but disregard this! I linked the oldest possible version of the modpack! I think that it is uploaded... I am so confused.
But to sum it up, don't download an update until I say that everything is all fixeded.
I made boo boo :P

**Comment by Xariyl** [comment]
Yesterday the FTB client downloaded v1.0.1 again and overwrote some settings. But nothing game-breaking. I did not modify the server at all, and my connection hasn't been affected.

No rush :)

Also, I think Vazkii nerfed the withering flower config. It used to be that setting it to -1 would disable the effect. But lately it hasn't been working. Setting it to 0 used to cause them to despawn almost instantly. So I'm not sure what's going on there.

**Comment by odenthefish** [comment]
Weee! the mudpack hath bean updatedith 
Server link: https://drive.google.com/folderview?id=0B7SE6kZCr1InN3JLZlRyX2hGVVE&usp=sharing
And we have artwork!

**Comment by odenthefish** [comment]
I'm going to start working on the next update, It is a few days overdue... 
Already I have a few things picked out to fix, 
Fix recipe conflict between Flan's mod AA gun barrel and steve's carts raw handle (crafting components) 
Add craftable gold coins from Custom NPCs ( I currently have them added as craftable on the server but that is kind of a silly way to do it)

Oh and did I mention Minetweaker? 

Do you have any requests for the next update? Any mods you want to see?

**Comment by odenthefish** [comment]
Oh and I keep forgetting, happy really late thanksgiving! I meant to tell you since thanksgiving but kept forgetting, I even forgot that it was thanks giving on thanks giving...
Sorry! And hope you had a goodone!

Edit: why didn't I post this in random stuff... who knows.

**Comment by Xariyl** [comment]
Thank you, and happy really really really really really late Thanksgiving to you too!

Minetweaker would be good, there are a few "quality of life" tweaks I like to do that I might throw in (I believe I can set some scripts server-side only so it won't mess with the modpack).

Any other mods I can think of at the moment would cause conflicts with the pack, I think; either deviates from the theme too much or would add unnecessary duplication of functionality. 

If I think of anything though I'll let you know. Did you have any others in mind?

**Comment by odenthefish** [comment]
I don't know of any mods, Sandriell asked me to add a mod that adds more food a whole back, like pam's harvest craft.
But I'm not a fan of everything that it adds, it seems too big and invasive.
Would you know of a good replacement?

**Comment by Xariyl** [comment]
I like AgriCraft. Allows crop-breeding to get more yield or more nutritional foods. Like bees/trees with Forestry and Fish with Mariculture.

I've never liked harvestcraft, though I can't really say exactly why. Something about it just bugs me. But I actually found AgriCraft fun.

What do you think of Mine & Blade? (can craft a shield to carry in your off hand)

**Comment by odenthefish** [comment]
Oh yea! I forgot I was going to add agricraft, but does it add new food? It's fine if it doesn't but I don't remember if it does or not.
Mine & Blade, oh yes, if I can get it working correctly it'll most likely be added, I used to play with it back in 1.6.4 but it had crashes pretty often, but that could have been something I was doing, I was pretty new to mods didn't even know how to read a crash report.

**Comment by Xariyl** [comment]
It does add some new foods and crops, you'll have to crossbreed some plants to get new crops, some crops depend on the soil they're on, etc.

Mine & Blade I've had decent success with in early 1.7.10 modpacks but it's been a while since I've used it. I was just trying to think of something to go along with the combat side of the pack.

Hm, what else? Progressive Automation, maybe, low-end simplified automation. Again not really necessary. I don't know.

**Comment by odenthefish** [comment]
Wow how did 6 days pass so quick? Anyway the modpack will be updated soon, so far this is the changelog 
V1.0.3 of Experiment 812390
Agricraft
Mine & Blade 
Minetweaker

Updated
Aromabackup
Bibliocraft
Botania
Buildcraft
Computercraft
Flaxbeard's steammod
Forestry
Gardenstuff
Journeymap *
Malisis core
Malisis doors
Mantle
Modularpowersuits
Morphius
Openmodularturrets
Pneumaticcraft
Simply jetpacks
Tconstruct
Wawla

Journey map may not be updated, it was acting odd, when I created a new world I spawned in a mega taiga, mega taigas contain coarse dirt, and the coarse dirt was showing as pure red.
I will do more testing but the red coarse dirt was fixed when I rolled back the version of Journey map.

**Comment by odenthefish** [comment]
Woo!!! after like 2 hours I figured out minetweaker! Man this is cool, and it's simple after I found out where to put the code stuff.
I fixed the Flan's mod AA gun barrel conflicting with the Steves' carts raw handle, and just about everything else is ready, the update will be soon!

**Comment by odenthefish** [comment]
About a month late the 1.0.3 update is complete! 

V1.0.3 of Experiment 812390
Agricraft
Mine & Blade 
Minetweaker
Mcicraft 1.0.7 core/farming/redstone (it adds a few cool things in the Redstone module such as a pulse extender \o/)

Updated
Aromabackup
Bibliocraft
Botania
Buildcraft
Computercraft
Flaxbeard's steammod
Forestry
Gardenstuff
Journymap
Malisis core
Malisis doors
Mantle
Modularpowersuits
Morphius
Openmodularturrets
Pneumaticcraft
Simply jetpacks
Tconstruct
Wawla

Added Minetweaker script for Flan's mod AA gun barrel so it doesn't conflict with Steves' carts Raw handle

It has been posted to the Feed The Beast thread so it should be up within the next day or so, if everything goes to plan :)

**Comment by Xariyl** [comment]
Sweet! 
Hm, I have to look at Mcicraft, I've not used that one.

**Comment by odenthefish** [comment]
I excluded the machines module simply because we have everything it offers and when you search "mci-craft" in nei it shows a massive technical block that takes up a huge amount of the screen  (by technical I mean a behind the scenes block used for rendering)

**Comment by odenthefish** [comment]
Yay! version 1.0.3 is live!
Server version here! https://drive.google.com/file/d/0B7SE6kZCr1InMzNCSWN6MzNCNlk/view?usp=sharing

This update is like a month late :P

**Comment by Xariyl** [comment]
Update in progress!

**Comment by odenthefish** [comment]
\o/ <=== my new favorite face

**Comment by Xariyl** [comment]
signed in... success! version 1.0.3 is now live on the server !

**Comment by Xariyl** [comment]
Are you getting an error message? I was online and I could see you trying to join but then you disconnect and I don't see any errors in the console/log.

**Comment by odenthefish** [comment]
Out dated forge I think :P I'm using a pre-existing instance and these stutters normally happen because I forget an obvious thing heh

**Comment by Xariyl** [comment]
Ah, crap... I forgot to mention, yeah, I had to update forge. Now using 1558.

forge-1.7.10-10.13.4.1558-1.7.10

**Comment by Xariyl** [comment]
Have you had any issues with the later versions of Logistic Pipes? I just that the server crashes, and the log looks like it's blaming LP. It's giving me an Out-of-Memory error, and I'm watching the server's CUI and RAM usage max out.

**Comment by odenthefish** [comment]
No I haven't had any issues with logistics pipes, though I haven't played with it much outside the server lately.
I'm using quite an old version, I forgot to update it to the latest update but before that I was using an old version because of (if I remember correctly) an issue with updating it.

I'll release a hot fix tonight, for now I'd just get the recommended version of LP for the server.
I need to finally see about updating project red as well...

Sorry about the problem! I'll try to get it fixed! Also could I have the crash log?

**Comment by odenthefish** [comment]
Small update for the server, it updates project red and LP right out of the box, the updated client is available on FTB as always :)
https://drive.google.com/file/d/0B7SE6kZCr1InS3YwTFNvUklOTTg/view?usp=sharing

**Comment by Xariyl** [comment]
Server updated!

**Comment by odenthefish** [comment]
\O/
  ||||
 / \

**Comment by odenthefish** [comment]
Work has begun on version 1.0.5 that was kind of 1.0.4 but it isn't listed as that because of my derpyness, I do believe that was my first use of "Derp" in an actual message.

I've been wanting to make a Scripts folder on github but I can't figure out how to make a folder for the queen, or king, or anyone I guess...
Like the scripts folder back in the Buildcraft next generation modpack.

What the update might, well update.

Updated Agricraft. Agrifun!
Updated Botania, who would have guessed? A new Botania update?
Updated Journey map. Better than vanilla maps.
Updated Modular powersuits. Wheee! I'm ferromagnetic man! We don't want Copy right claims! ( Yeah right this is totally offical) 
Updated Open blocks, Openmods lib. I really don't have much of a joke for this one...
Updated Open Modular turrets. Turrety! 
Updated Roguelike dungeones oooOOOOoooooo! Spooky ghost sound! Why'd that make me think of cheesy ghost sounds?
Updated Schematica. Skiimatica for all you ski-matic pronouncers.
Updated Tinkers' Mechworks. Tinkery!
Updated Project red modules. <Insert silly little stupid joke here>
Updated Forestry. Bees everywhere! EVERYWHERE.

**Comment by Xariyl** [comment]
It's been a while since I've done it, but to create a new folder you'll first need to create your own Branch. Within your branch, create a New File. In the space for the filename, you can add the folder (type "/scripts/") and then the name of the file you're creating.

When you're done with all the changes you want to make -- scripts, configs, whatever -- you commit the branch and then do a pull-request to merge your changes into the master. I'll get the notice and approve it.

If I remember correctly :smile:

**Comment by odenthefish** [comment]
Thanks! The modpack is going to be uploaded to FTB today!

**Comment by odenthefish** [comment]
The modpack will actually be submitted today, sorry about the delay!

**Comment by odenthefish** [comment]
Again sorry about the delay, the modpack will be submitted tonight, really!
Sorry about how long it's taken, I'm in the final stages of testing.

**Comment by odenthefish** [comment]
The modpack is finally finalized and should be submitted in about an hour, I'm so sorry about the wait time and all the confusion, I have no excuse for not getting it done, I just haven't been in the groove to finish it, I normally feel somewhat energized to get a new update out, but not this update.
Anyways, this update doesn't change much, it updates a few mods and adds mod tweaker  (\0/) and a couple of configs have been changed  (the darkness of hardcore darkness changes with the phases of the moon!)

**Comment by Xariyl** [comment]
No rush, no worries. I've been out of it lately too. When I do play, I've been testing out some 1.8.9 mods, but generally I've been playing other games. I kind of need a break from Minecraft in general, I think. So no need to apologize. I'll get the server updated soon as everything is ready.

**Comment by odenthefish** [comment]
Updated to 1.0.5! 
Server: https://drive.google.com/folderview?id=0B7SE6kZCr1InM1FCZWdnZ05tUWM&usp=sharing

**Comment by Xariyl** [comment]
Server files updated.
- Forge version updated to " forge-1.7.10-10.13.4.1614-1.7.10-universal ".
- The ProjectRed.cfg file that's included with the server files seemed to have gotten overwritten by the mod jar when the server was restarted. I had to manually update the "General Settings: Enchantment: Fuel Efficiency" value -- default was 80, I changed it to 180.

Looked like everything loaded okay. I haven't had a chance to go online to verify everything yet. I'll be out of town for a couple of weeks and won't be bringing a gaming computer with me. But all files are loaded and the server is running.

**Comment by odenthefish** [comment]
Lol, that's the same number I changed it to, I fixed the ID in the config before uploading it, but the server rewrote it, I'll release an update to fix it ASAP, after all I'm sure Botania's updated again.

**Comment by Xariyl** [comment]
Yeah, I looked at the downloaded file and saw it was set to 180, but the file on the server had a different layout and had reset the value to 80. So looks like other things had changed and the mod jar is designed to overwrite the config.
